packages/compass-crud/src/components/readonly-filter.tsx

```diff
@@ -50,20 +51,19 @@ export function ReadonlyFilter({
   filterQuery,
 }: ReadonlyFilterProps) {
   return (
-    <TextInput
-      id="readonly-filter"
-      data-testid="readonly-filter"
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore the label can be any component, but it's weirdly typed to string
-      label={
-        <QueryLabel
-          label={queryLabel}
-          tooltip="Return to the Documents tab to edit this query."
-        />
-      }
-      disabled={true}
-      value={filterQuery}
-      className={textInputStyles}
-    />
+    <>
+      <QueryLabel
+        label={queryLabel}
+        tooltip="Return to the Documents tab to edit this query."
+      />
+      <KeylineCard
+        id="readonly-filter"
+        data-testid="readonly-filter"
+        disabled={true}
+        className={textInputStyles}
+      >
+        <code>{filterQuery}</code>
```

Comment:
This is not using leafygreen monospace font styles

Reply:
fixed, thanks for pointing it out